README.md

```diff
@@ -39,7 +41,7 @@ We encourage you to compile the documentation locally on your computer prior to
     C:\> choco install python
     ```
 
-4. Restart cmd or refresh with the command:
+4. Restart cmd as administrator or refresh with the command:
```

Comment:
Have you confirmed this? On Linux you do not need elevated privileges for installing pip packages.

Reply:
Yuppers, but only on Windows. It gives an error that doesn't make it clear that it's an administrator issue. I've updated it to be more clear.

Reply:
These instructions are for Windows anyways, so info that this only needs administrative privileges on Windows seems unnecessary.
I just wanted to make sure that you confirmed it actually needed them.

Reply:
Ah, I might as well revise it to be clear in general lol since I assume the same commands past PowerShell work the same on Linux?

Reply:
It depends on the Linux distribution, but generally no. Also some Linux distributions will need additional packages.